packages/adapters-library/src/adapters/aave-v2/products/a-token/aaveV2ATokenAdapter.ts

```diff
@@ -10,19 +10,21 @@ import { ProtocolDataProvider } from '../../contracts'
 export class AaveV2ATokenPoolAdapter extends AaveBasePoolAdapter {
   productId = 'a-token'
 
+  adapterSettings = {
+    enablePositionDetectionByProtocolTokenTransfer: true,
+    includeInUnwrap: true,
+  }
+
   getProtocolDetails(): ProtocolDetails {
     return {
       protocolId: this.protocolId,
       name: 'Aave v2 AToken',
       description: 'Aave v2 defi adapter for yield-generating token',
       siteUrl: 'https://aave.com/',
       iconUrl: 'https://aave.com/favicon.ico',
-      positionType: PositionType.Lend,
+      positionType: PositionType.Supply,
       chainId: this.chainId,
       productId: this.productId,
-      assetDetails: {
-        type: AssetType.StandardErc20,
```

Comment:
Ive dropped asset type because we dont have a use for it and I dont want it to confuse us, if its gone we know no logic is using it